src/styles/blocks/page.scss

```
@@ -0,0 +1,92 @@
+.page {
+  cursor: url(../images/cursor.svg), auto;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+    padding: 0;
+    
+    &--with--menu {

```

.page__body--with-menu - the class name
```suggestion
    &--with-menu {

```